data/SugarBean.php

```diff
@@ -5515,4 +5494,28 @@ public function create_export_query($order_by, $where)
     {
         return $this->create_new_list_query($order_by, $where, array(), array(), 0, '', false, $this, true, true);
     }
+
+
+    /**
+     * @param $isUpdate
+     */
+    public function auditBean($isUpdate){
+        if ($this->is_AuditEnabled() && $isUpdate && isset($this->fetched_row))
+        {
+            $auditDataChanges = $this->db->getAuditDataChanges($this);
+
+            if (!empty($auditDataChanges) && is_array($auditDataChanges))
+            {
+                foreach ($auditDataChanges as $change)
+                {
+                    $this->db->save_audit_records($this, $change);
+                    $this->fetched_row[$change['field_name']] = $change['after'];
+                }
+            }
+        }
+        else
+        {
+            $GLOBALS['log']->debug('Auditing: Retrieve was not called, audit record will not be created.');
```

Comment:
should only log it if $isUpdate is true and !isset($this->fetched_row), maybe split $this->is_AuditEnabled() && $isUpdate into a separate if statement